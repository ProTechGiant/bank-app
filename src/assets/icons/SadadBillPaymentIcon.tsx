import { Path, Svg } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function SadadBillPaymentIcon({ color = "#01322A", height = 24, width = 24 }: IconProps) {
  return (
    <Svg xmlns="http://www.w3.org/2000/svg" width={width} height={height} viewBox="0 0 24 24" fill="none">
      <Path
        d="M2.80005 22.1V1.92505C2.81672 1.92505 2.86272 1.92905 2.93805 1.93705C3.01272 1.94572 3.09172 1.99172 3.17505 2.07505L3.97505 2.87505C4.07505 2.99172 4.19605 3.04572 4.33805 3.03705C4.47938 3.02905 4.60838 2.97505 4.72505 2.87505L5.47505 2.10005C5.59172 1.98338 5.72105 1.92505 5.86305 1.92505C6.00438 1.92505 6.13338 1.97505 6.25005 2.07505L7.02505 2.87505C7.12505 2.97505 7.25005 3.02505 7.40005 3.02505C7.55005 3.02505 7.67505 2.97505 7.77505 2.87505L8.55005 2.07505C8.66672 1.97505 8.79605 1.92505 8.93805 1.92505C9.07938 1.92505 9.20838 1.97505 9.32505 2.07505L10.1 2.87505C10.2 2.97505 10.325 3.02505 10.475 3.02505C10.625 3.02505 10.75 2.97505 10.85 2.87505L11.625 2.07505C11.725 1.97505 11.85 1.92505 12 1.92505C12.15 1.92505 12.275 1.97505 12.375 2.07505L13.15 2.87505C13.2667 2.97505 13.396 3.02505 13.538 3.02505C13.6794 3.02505 13.8084 2.97505 13.925 2.87505L14.7 2.07505C14.8 1.97505 14.925 1.92505 15.075 1.92505C15.225 1.92505 15.35 1.97505 15.45 2.07505L16.225 2.87505C16.3417 2.97505 16.471 3.02505 16.613 3.02505C16.7544 3.02505 16.8834 2.97505 17 2.87505L17.775 2.07505C17.875 1.97505 18 1.92505 18.15 1.92505C18.3 1.92505 18.425 1.97505 18.525 2.07505L19.3 2.87505C19.4 2.97505 19.525 3.02505 19.675 3.02505C19.825 3.02505 19.95 2.96672 20.05 2.85005L20.8251 2.10005C20.9251 2.00005 21.0084 1.94572 21.0751 1.93705C21.1417 1.92905 21.1834 1.92505 21.2001 1.92505V22.1C21.1834 22.1 21.1417 22.0957 21.0751 22.087C21.0084 22.0791 20.9251 22.0334 20.8251 21.9501L20.05 21.175C19.95 21.075 19.825 21.0207 19.675 21.012C19.525 21.004 19.4 21.0584 19.3 21.175L18.525 21.9501C18.425 22.0501 18.3 22.1 18.15 22.1C18 22.1 17.875 22.0501 17.775 21.9501L17 21.175C16.8834 21.0584 16.7544 21 16.613 21C16.471 21 16.3417 21.0584 16.225 21.175L15.45 21.9501C15.35 22.0501 15.225 22.1 15.075 22.1C14.925 22.1 14.8 22.0501 14.7 21.9501L13.925 21.175C13.8084 21.0584 13.6794 21 13.538 21C13.396 21 13.2667 21.0584 13.15 21.175L12.375 21.9501C12.275 22.0501 12.15 22.1 12 22.1C11.85 22.1 11.725 22.0501 11.625 21.9501L10.85 21.175C10.7334 21.0584 10.6044 21 10.463 21C10.321 21 10.1917 21.0584 10.075 21.175L9.30005 21.9501C9.20005 22.0501 9.07505 22.1 8.92505 22.1C8.77505 22.1 8.65005 22.0501 8.55005 21.9501L7.77505 21.175C7.65838 21.0584 7.52938 21 7.38805 21C7.24605 21 7.11672 21.0584 7.00005 21.175L6.22505 21.9501C6.12505 22.0501 6.00005 22.1 5.85005 22.1C5.70005 22.1 5.57505 22.0501 5.47505 21.9501L4.70005 21.175C4.60005 21.0584 4.47505 21.004 4.32505 21.012C4.17505 21.0207 4.05005 21.075 3.95005 21.175L3.17505 21.9501C3.07505 22.0334 2.99172 22.0791 2.92505 22.087C2.85838 22.0957 2.81672 22.1 2.80005 22.1ZM7.12505 17.05H16.875C17.175 17.05 17.425 16.95 17.625 16.75C17.825 16.55 17.925 16.3 17.925 16C17.925 15.7 17.825 15.45 17.625 15.25C17.425 15.05 17.175 14.95 16.875 14.95H7.12505C6.82505 14.95 6.57505 15.05 6.37505 15.25C6.17505 15.45 6.07505 15.7 6.07505 16C6.07505 16.3 6.17505 16.55 6.37505 16.75C6.57505 16.95 6.82505 17.05 7.12505 17.05ZM7.12505 13.05H16.875C17.175 13.05 17.425 12.95 17.625 12.75C17.825 12.55 17.925 12.3 17.925 12C17.925 11.7 17.825 11.45 17.625 11.25C17.425 11.05 17.175 10.95 16.875 10.95H7.12505C6.82505 10.95 6.57505 11.05 6.37505 11.25C6.17505 11.45 6.07505 11.7 6.07505 12C6.07505 12.3 6.17505 12.55 6.37505 12.75C6.57505 12.95 6.82505 13.05 7.12505 13.05ZM7.12505 9.05005H16.875C17.175 9.05005 17.425 8.95005 17.625 8.75005C17.825 8.55005 17.925 8.30005 17.925 8.00005C17.925 7.70005 17.825 7.45005 17.625 7.25005C17.425 7.05005 17.175 6.95005 16.875 6.95005H7.12505C6.82505 6.95005 6.57505 7.05005 6.37505 7.25005C6.17505 7.45005 6.07505 7.70005 6.07505 8.00005C6.07505 8.30005 6.17505 8.55005 6.37505 8.75005C6.57505 8.95005 6.82505 9.05005 7.12505 9.05005ZM5.07505 19.025H18.925V4.97505H5.07505V19.025Z"
        fill={color}
      />
    </Svg>
  );
}
