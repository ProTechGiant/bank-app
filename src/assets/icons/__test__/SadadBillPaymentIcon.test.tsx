import { create } from "react-test-renderer";

import { SadadBillPaymentIcon } from "../SadadBillPaymentIcon";

describe("icon/SadadBillPaymentIcon", () => {
  it("renders a SadadBillPaymentIcon", () => {
    const result = create(<SadadBillPaymentIcon />);

    expect(result).toMatchSnapshot();
  });
});
