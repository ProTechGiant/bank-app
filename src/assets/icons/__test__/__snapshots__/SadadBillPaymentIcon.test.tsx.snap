// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/SadadBillPaymentIcon renders a SadadBillPaymentIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={24}
  vbWidth={24}
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGGroup
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      name="outlined_flag"
    >
      <RNSVGPath
        d="M13.5 5.5L12.5 3.5H4.5V20.5H6.5V13.5H11.5L12.5 15.5H19.5V5.5H13.5ZM17.5 13.5H13.5L12.5 11.5H6.5V5.5H11.5L12.5 7.5H17.5V13.5Z"
        fill={
          {
            "payload": 4294931730,
            "type": 0,
          }
        }
        name="Vector"
        propList={
          [
            "fill",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGGroup>
</RNSVGSvgView>
`;
